src/whatsnew/2.1.rst

```diff
@@ -20,11 +20,162 @@
+  * :ghissue:`752`: Allow Mango $in queries without requiring the index to
+    use an array
+  * (multiple) Additional debugging utilities have been added.
+  * (multiple) Hot code upgrades from 2.0 -> 2.1.1 are now possible.
+  * (multiple) Improvements to the test suite have been made.
+  * :ghissue:`765`: Mango ``_explain`` now includes view parameters as requested
+    by the user.
+  * :ghissue:`653`: `_show` and `_list` should now work for admin-only
+    databases such as ``_users``.
+  * :ghissue:`807`: Mango index selection should occur only once.
+  * :ghissue:`804`: Unhandled Mango errors are now logged.
+  * :ghissue:`659`: Improve accuracy of the ``max_document_size`` check.
+  * :ghissue:`817`: Invalid Base64 in inline attachments is now caught.
+  * :ghissue:`825`: Replication IDs no longer need to be URL encoded when
+    using the ``_scheduler/jobs/<job_id>`` endpoint.
+  * :ghissue:`838`: Do not buffer rexi messages to disconnected nodes.
+  * :ghissue:`830`: The stats collection interval is now configurable in
+    an ini file, not in the application context. The default value is 10,
+    and the setting is reloaded every 600 seconds.
+  * :ghissue:`812`: The ``/{db}`` endpoint now includes a ``cluster`` block
+    with the database's ``q``, ``n``, and default ``w`` and ``r`` values.
+    This supplements the existing ``/{db}/_shards`` and ``/{db}/_shards/{id}``
+    detailed information on sharding and quorum.
+  * :ghissue:`810`: The replicator scheduler crashed counter gauge more
+    reliably detects replication crashes by reducing the default number
+    of retries from 10 to 5 (reducing the duration from 4 mins to 8 secs).
+  * :issue:`3288`: Tolerate mixed clusters for the upcoming pluggable
+    storage engine work.
+  * :ghissue:`839`: Mango python tests now support Python 3 as well as 2.
+  * :ghissue:`845`: A convenience ``remsh`` script has been added to support
+    live debugging of running systems.
+  * :ghissue:`846`: Replicator logging is now less verbose and more informative
+    when replication terminates unexpectedly.
+  * :ghissue:`797`: Reduce overflow errors are now returned to the client,
+    allowing views with a single bad reduce to build while not crushing the
+    server's RAM usage.
+  * :ghissue:`881`: Mango now allows match on documents where the indexed
+    value is an object if a range query is issued. Previously, query results
+    might change in the presence of an index, and operators/selectors which
+    explicitly depend on a full index scan (such as ``$exists``) would not
+    return a complete result set.
+
 .. _release/2.1.0:
 
 Version 2.1.0
 =============
 
+* :ghissue:`808`: Mango now supports
+  :ref:`partial indexes <find/partial_indexes>`. Partial indexes allow
+  documents to be filtered at indexing time, potentially offering
+  significant performance improvements for query selectors that don't map
+  cleanly to a range query on an index.
+
```

Comment:
partial indexes is a new feature for `2.1.1`

Reply:
WHoops, wrong section! Good catch